packages/react-components/src/atoms/Datetime.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+interface DatetimeProps {
+  readonly date: Date;
+}
+
+const Datetime: React.FC<DatetimeProps> = ({ date }) => {
```

Comment:
This should probably be just a function? I could see this causing problems with components that expect text as children